velox/common/base/Exceptions.h

```diff
@@ -172,14 +172,20 @@ std::string errorMessage(fmt::string_view fmt, const Args&... args) {
         exception,                                                       \
         typename ::facebook::velox::detail::VeloxCheckFailStringType<    \
             decltype(message)>::type>(veloxCheckFailArgs, message);      \
-  }
+  }                                                                      \
+  static_assert(                                                         \
+      true,                                                              \
+      "") // The assertion is needed to force macro calls to end with `;`.
```

Comment:
Will `static_assert` be optimized out in the opt build?

Reply:
yes I think it doesn't survive runtime